### ISSUE TITLE
Bitdefender Blocking the script running through Powershell 

### ISSUE THREAD

**Issue by Temoi89** [ChrisTitusTech/winutil#2433]
![powershell_DFgvAYgN7r](https://github.com/user-attachments/assets/33a40406-3bef-458f-b634-71496f28ce19)

**Comment by nowrap** [comment]
In the notification area it says:
Trojan.GenericKD.73614837 detected

**Comment by YusufKhalifadev** [comment]
@Temoi89 @nowrap This happens because the script is not signed from Microsoft so any antivirus detects it as Malware but it's not so to fix this you need to turn off your antivirus and then run the script.

btw it's not signed because the prices for signing a powershell script are too high per year 
here's the source: https://shop.certum.eu/data-security/code-signing-certificates/certum-ev-code-sigining.html

**Comment by Temoi89** [comment]
Will the executable run in the admin mode then. Hope that's signed. Thanks
a lot anyway, for the response. Meanwhile, I'll try contacting Bitdefender
to mark it as a false positive.

**Comment by FatBastard0** [comment]
I have the same issues with Bitdefender!  I have to turn if off to use.

**Comment by Chiramisu** [comment]
This seems related to #2484 (See [my comment](https://github.com/ChrisTitusTech/winutil/issues/2484#issuecomment-2276457651) as I'm also using the shortcut from the `autounattend.xml` install), but it seems the issue is that you're running an undocumented command as it's supposed to be `irm https://christitus.com/win | iex` (not `iwr`). When I run your command, I also get an error, but again, this is the wrong command (see the [README](https://github.com/ChrisTitusTech/winutil?tab=readme-ov-file#stable-branch-recommended)).

The difference is that [Invoke-RestMethod](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7.4) "Sends an HTTP or HTTPS request to a RESTful web service" (correct), whereas [Invoke-WebRequest](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7.4) "Gets content from a web page on the internet" and the endpoint `https://christitus.com/win` is not a webpage, but rather serves up the `.ps1` script file directly.

In other words, "you're doing it wrong." 😁

**Comment by FatBastard0** [comment]
Well thank you for telling me I was doing wrong like talk about customers
service now what time you do it right
I would like to thank you in advance for any assistance that you can
provide in this matter. Please feel free to contact me at the below listed
address, Telephone and/or email.




Richard K. L'Italien
Disable Veteran of the US Army
The Fat Bastard
5637 Monte Corita Circle
Citrus Heights California 95621
(916) 915-3883 <http://voice.google.com/calls?a=nc,%2B19169153883>
<http://voice.google.com/calls?a=nc,%2B19169153883> Voice
(916) 943 4967 <http://voice.google.com/calls?a=nc,%2B19169434967>
<http://voice.google.com/calls?a=nc,%2B19167453437>

******************************************************************************
This communication (including any attachments) may contain privileged or
confidential information intended for a specific individual and purpose,
and is protected by law.  If you are not the intended recipient, you should
delete this communication and/or shred the materials and any attachments and
are hereby notified that any disclosure, copying, or distribution of this
communication, or the taking of any action based on it, is strictly
prohibited.

Thank you.

**Comment by Marterich** [comment]
Nothing else to do at the moment /close
